samples/runtime/autosubstitution/AutoSubstitution.qs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Project {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Targeting;
+
+    operation ClassicalSWAP(a : Qubit, b : Qubit) : Unit is Adj + Ctl {
```

Comment:
Strictly speaking, this operation name isn't a verb, but may fall under the same shorthand exception that "SWAP" itself uses.